### DISCUSSION TITLE
Considerazioni varie

### DISCUSSION THREAD

**Discussion by Xenon-2** [fdivitto/FabGL#251] [General]
Ciao Fabrizio.

Complimenti per il lavoro di cui sono venuto a conoscenza solo la settimana scorsa.

Ti vorrei segnallare alcune cose:

**Velocita di emulazione:** il sistema è troppo veloce rispetto al pc ibm, sarebbe opportuno poter selezionare la velocità di emulazione, per mio uso ho aggiunto la selezione in fase di configurazione e sul "ALT+PRTSC", altrimenti giochi come alleycat o big top sarebbero ingiocabili. In realtà ho trovato una soluzione che media perchè le prestazioni della scheda video sono molto scadenti.

**Suono**: non sempre funziona. Ad esempio su bigtop non funziona per niente. Pensavo fosse dovuto alla gestione diretta delle porte,  ma ho fatto delle prove in assembler e funziona. se hai modo dovresti provare con il gioco BIGTOP.

**Cambio floppy**: Se cerchi di cambiare il floppy ed associare una immagine che si trova in una directory, non funziona. questo perchè non conserva il path completo e cerca di caricarlo dalla root della SD.

**Cambio BIOS**: Sarebbe interessante poter caricare versioni diverse. Ho provato ma non funziona.

**Seriale o lan via WIFI**: interessante per la connettività. 

**Scheda video**: selezionabile in fase di configurazione(Hercules/CGA). A tal proposito io aggiungere un ulteriore esp32 che funga solo da scheda video(opzionale). Questo perchè è il componente più debole rispetto alle prestazioni del pc IBM.

**Floppy esterno**: drawbridge ( https://github.com/RobSmithDev/ArduinoFloppyDiskReader ) oppure usb. darebbe un grande realismo. 

**Emulazione**: molto buona. con DRDOS si blocca molto spesso, cosa che accade raramente con ms-dos.

Le mie prove sono state fatto con la TTGO vga32 1.4.

Grazie, Gianni.

**Comment by fdivitto** [comment]
Ciao Gianni,
prima di tutto grazie per i test ed i suggerimenti. Saranno elementi preziosi quando potrò affrontare determinati aspetti di questa emulazione.
Ho spostato la Issue in Discussions in quanto troppo vasta perchè io possa fare un fix e poi chiuderla.

Grazie,
Fabrizio

PS: dovremmo utilizzare l'Inglese perchè ogni osservazione fatta qui potrebbe essere utile ad altre persone.

**Reply by Xenon-2** [reply]
Thank you very much.
Sorry, all other conversations will be in english.

**Comment by gnokketto** [comment]
write here.. the LIST tab not working (the site 8bits is down) its possibile download from other site ?
